provider/pkg/provider/command.go

```diff
@@ -54,6 +56,20 @@ func (c *command) RunCreate(ctx context.Context, host *provider.HostClient, urn
 	return id, err
 }
 
+//
+func (c *command) RunUpdate(ctx context.Context, host *provider.HostClient, urn resource.URN) (string, error) {
+	if c.Update != nil {
+		stdout, stderr, id, err := c.run(ctx, *c.Update, host, urn)
```

Comment:
Should we dereference Update or not (we don't with Create below)

Reply:
We should, because update is *string, similar to the delete field, where create is a string.